lightly/api/api_workflow_datasources.py

```diff
@@ -590,4 +587,30 @@ def get_custom_embedding_read_url(
         return self._datasources_api.get_custom_embedding_file_read_url_from_datasource_by_dataset_id(
             self.dataset_id,
             filename,
-        )
\ No newline at end of file
+        )
+
+    def list_datasource_permissions(
+        self,
```

Comment:
Is it OK that the function uses `client.dataset_id` or should it get the dataset id as the input?

Reply:
Yes that is ok, most functions assume that `client.dataset_id` is used.